requirements-dev.txt

```diff
@@ -16,6 +16,7 @@
 #
 black==19.3b0
 coverage==4.5.3
+cockroachdb==0.3.3
```

Comment:
This needs to be moved to `extras_require` in `setup.py`.

Reply:
done